corehq/apps/translations/management/commands/translate_po_files.py

```
@@ -0,0 +1,672 @@
+
+class PoTranslationFormat(TranslationFormat):
+    """
+    Translation format for PO files. The class expects gettext installed in the system.
+    As it uses gettext's msgfmt command to check for errors in the PO file.
+    """
+
+    def __init__(self, file_path):
+        self.file_path = file_path
+        self.translation_obj_map = {}
+
+    def load_input(self, po_objects=[]):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)